src/main/java/logisticspipes/request/RequestTreeNode.java

```diff
@@ -620,13 +605,10 @@ private static List<Pair<ICraftingTemplate, List<IFilter>>> getCrafters(
                     if (craftable != null) {
                         for (IFilter filter : r.filters) {
                             if (filter.isBlocked() == filter.isFilteredItem(craftable.getResultItem())
-                                    || filter.blockCrafting()) {
-                                continue;
```

Comment:
Why was 'continue' deleted?

Reply:
because it's the last statement of the loop, question is : why is there a loop with just an empty if statement ?